plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -301,6 +301,16 @@ private class DokkaDescriptorVisitor(
         }
     }
 
+    private fun ClassDescriptor.getEnumDescriptorsWithKind(): DescriptorsWithKind {
+        val descriptorsWithKind = this.unsubstitutedMemberScope.getDescriptorsWithKind()
+        val staticScopeForKotlinEnum = (this.staticScope as? StaticScopeForKotlinEnum) ?: return descriptorsWithKind
+
+        // synthetic values() and valueOf() functions are not present among average class functions
+        val enumSyntheticFunctions = staticScopeForKotlinEnum.getContributedDescriptors { true }
```

Comment:
Are you sure it will contain only functions?! 
Anyway it is not a blocker to merge the PR.

`staticScope` is `MemberScope` too.
Yet another implementation:
```kt
val descriptorsFromMemberScope = unsubstitutedMemberScope.getDescriptorsWithKind()
val descriptorsFromStaticScope = staticScope.getDescriptorsWithKind()
returns descriptorsFromMemberScope  + descriptorsFromStaticScope // a plus operator - TODO
```

Reply:
The return type for `getContributedDescriptors` is `SimpleFunctionDescriptor`, so it should be just functions, yeah

I thought about what you suggested, but it seems like calling `getDescriptorsWithKind` on `staticScope` would add some overhead and require writing even more code to support it, so I think it's best to keep it simple